README.md

```diff
@@ -36,7 +32,7 @@ After installing the SDK according to the official documentation, the libs and h
 
 If you used Conan to retrieve your dependencies, you can use the following commands to build the project:
 
-    cmake -S . -B build -DCMAKE_MODULE_PATH:PATH=/path/to/modules
+    cmake -S . -B build -DCMAKE_TOOLCHAIN_FILE=generators/conan_toolchain.cmake
```

Comment:
This command is not working as is on Windows. Further explanations might be required.

Reply:
Switched to `--preset`